plugin-maven/src/main/java/com/diffplug/spotless/maven/AbstractSpotlessMojo.java

```diff
@@ -71,6 +71,9 @@
 	@Parameter
 	private LicenseHeader licenseHeader;
 
+	@Parameter
+	private Format format;
```

Comment:
Existence of both `LicenseHeader` and `Format` field permits the following configuration:

```
<configuration>
  <licenseHeader>...</licenseHeader>
  <format>
    <licenseHeader>...</licenseHeader>  
  </format>
</configuration>
```

which seems invalid because `<format>` is like a global-default.

Reply:
In the gradle plugin, `<format>` is not a global default.  It's a way for using manual steps like `replace` and `indent` to do specific formatting on specific files which aren't handled well by language-specific stuff like `java` and `scala`.

Reply:
I don't understand what you mean. It's 100% the same as with `global` and `<java>`, isn't it?

Reply:
Please never mind all my comments about `<format>`. I did not understand what is it.